odoo/addons/base/tests/test_misc.py

```diff
@@ -468,3 +477,26 @@ def test_readonly_dict(self):
             d.update({'baz': 'xyz'})
         with self.assertRaises(TypeError):
             dict.update(d, {'baz': 'xyz'})
+
+
+class TestUrlValidate(BaseCase):
+    def test_url_validate(self):
+        for case, truth in [
```

Comment:
subtest? always nice even if they'll most likely never fail anyway

Reply:
Same here, it was solved in second commit :)